rm-datasource/src/main/java/io/seata/rm/datasource/sql/SQLRecognizerGroup.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.rm.datasource.sql;
+
+import com.alibaba.druid.sql.ast.SQLStatement;
+
+/**
+ * The interface SQLRecognizerGroup
+ *
+ * @author: Zhibei Hao丶
+ * @date: 2019/11/8 17:39
+ * @version: V1.0
+ */
+public interface SQLRecognizerGroup
```

Comment:
SQLOperateRecognizer

Reply:
rename to SQLOperateRecognizer or SQLOperateRecognizerGroup?

Reply:
I think the word `group` is rather vague and literally thinks it is a collection.

Reply:
Indeed, "group" is not appropriate. I also have several alternatives, such as  UndoExecutorHolder, UndoExecutorContainer or just UndoExecutor(s) , but I'm not sure which one is the most suitable. Could you give me some advise? Thanks!

Reply:
Generally, it does not represent its plural in the class name or interface name.The interface name is an abstract representation of the behavior

Reply:
OK,Thanks,I've changed its name To UndoExecutorHolder.